src/tools/spake2p/Cmd_GenVerifier.cpp

```diff
@@ -334,10 +372,14 @@ bool Cmd_GenVerifier(int argc, char * argv[])
             return false;
         }
 
-        // On the next iteration the PIN Code and Salt will be randomly generated.
-        gPinCode        = chip::kSetupPINCodeUndefinedValue;
+        gPinCode = GetNextPinCode();
```

Comment:
Need a comment what happens if the file with PINs wasn't provided. 
The PIN will be randomly generated in this case?

Reply:
Comment added